### PR TITLE
Clarify note about configuring Fastly with production settings

### LINKED ISSUE #8261: Clarify Fastly set up info about DNS configuration for production
# General issue

## Description:

The following text is mentioned in the [Fastly set up](https://devdocs.magento.com/cloud/cdn/configure-fastly.html) topic towards the top of the page:

```
When you are ready to launch your Production site, you must update your DNS configuration to point your production domains to the Fastly service and complete additional configuration tasks. See Launch checklist.
```

Logically speaking, the development settings would be configured first, then the Production settings would later be configured prior to launching.

## Possible solutions:

This section about configuring the Production settings Fastly should be moved under the `development settings` towards the bottom of the page.

### PR DESCRIPTION
**Pull request by chiranjeevi-cj**
## Purpose of this pull request

The following text in the [Fastly Setup](https://devdocs.magento.com/cloud/cdn/configure-fastly.html) topic in the _Cloud Guide_ mentioned towards the top of the page:

```
When you are ready to launch your Production site, you must update your DNS configuration to point your production domains to the Fastly service and complete additional configuration tasks. See Launch checklist.
Logically speaking, the development settings would be configured first, then the Production settings would later be configured prior to launching.

```
This pull request (PR) moves This section about configuring the Production settings Fastly under the development settings towards the bottom of the page.

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/cdn/configure-fastly.html

## Links to Magento source code

-  Fixes #8261

### CODE REVIEW

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
This is a more logical place for this statement to go - good call. 
However, as this is now right at the bottom of the page and underneath an alert box, it might get overlooked. May I suggest this be placed in an Information Box to make it stand out?

Ignore - already done!

**Comment by meker12** [comment]
@chiranjeevi-cj  Thanks for the PR to clarify the Fastly DNS configuration section.

I moved the info about updating DNS for production to the section on DNS configuration changes and made it an important note.  Can you please rebase your PR to the master branch?  This update is not related to the Magento 2.4.2 release.  Thanks!

File: src/cloud/cdn/configure-fastly.md [hunk 1]
```diff
@@ -173,6 +171,7 @@ Check with your [DNS registrar](https://lookup.icann.org/) for information about
 -  Provide a list of top-level and subdomains for each environment to your Customer Technical Advisor (CTA) or submit them in a Magento Support ticket.
 -  Wait for confirmation that the specified domains have been added to your Cloud environments.
 -  On Starter projects, add the domains to your Fastly service configuration. See [Manage domains]({{ site.baseurl }}/cloud/cdn/configure-fastly-customize-cache.html#manage-domains).
+-  For information about updating the DNS configuration, check with your [DNS registrar](https://lookup.icann.org/) for information about updating the DNS configuration.
```
Comment (BarnyShergold) [inline src/cloud/cdn/configure-fastly.md:174]: The phrase "For information about updating the DNS configuration" is repeated in the same sentence.

```suggestion
-  For information about updating the DNS configuration, check with your [DNS registrar](https://lookup.icann.org/) on the correct method for your domain service.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

File: src/cloud/cdn/configure-fastly.md (hunk 1 above)
Comment (meker12) [inline src/cloud/cdn/configure-fastly.md:174]:
```suggestion
-  For information about updating the DNS configuration, check with your [DNS registrar](https://lookup.icann.org/).
```

File: src/cloud/cdn/configure-fastly.md [hunk 2]
```diff
@@ -173,6 +171,7 @@ Check with your [DNS registrar](https://lookup.icann.org/) for information about
 -  Provide a list of top-level and subdomains for each environment to your Customer Technical Advisor (CTA) or submit them in a Magento Support ticket.
 -  Wait for confirmation that the specified domains have been added to your Cloud environments.
 -  On Starter projects, add the domains to your Fastly service configuration. See [Manage domains]({{ site.baseurl }}/cloud/cdn/configure-fastly-customize-cache.html#manage-domains).
+-  For information about updating the DNS configuration, check with your [DNS registrar](https://lookup.icann.org/).
```
Comment (meker12) [inline src/cloud/cdn/configure-fastly.md:174]:
```suggestion
-  For information about updating the DNS configuration, check with your [DNS registrar](https://lookup.icann.org/) for the correct method for your domain service.
```

File: src/cloud/cdn/configure-fastly.md (hunk 1 above)
Comment (meker12) [reply src/cloud/cdn/configure-fastly.md]: I realized this mistake before seeing your suggestion.  Please review the updated version. Thx.

**Review by BarnyShergold — APPROVED** [review]
Awesome!

**Comment by chiranjeevi-cj** [comment]
@meker12 @BarnyShergold Should I need to rebase this branch to master?

**Comment by chiranjeevi-cj** [comment]
@meker12 Rebased this branch to master. Thank you for assisting me.

**Review by meker12 — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Pull request merged**